src/models/activityReport.js

```diff
@@ -32,6 +32,13 @@ module.exports = (sequelize, DataTypes) => {
       ActivityReport.hasMany(models.NextStep, { foreignKey: 'activityReportId', as: 'specialistNextSteps' });
       ActivityReport.hasMany(models.NextStep, { foreignKey: 'activityReportId', as: 'recipientNextSteps' });
       ActivityReport.hasMany(models.ActivityReportApprover, { foreignKey: 'activityReportId', as: 'approvers', hooks: true });
+      ActivityReport.hasMany(models.ActivityReportGoal, { foreignKey: 'activityReportId', as: 'activityReportGoals' });
+      // ActivityReport.belongsToMany(models.Goal, {
```

Comment:
Do we want this commented out?

Reply:
I think is was breaking something, we can try uncommenting it.

Reply:
I lied, figured it out